runtime/themes/base16_transparent.toml

```diff
@@ -53,19 +54,19 @@
 "markup.select" = { fg = "magenta" }
 
 "diff.plus" = "light-green"
-"diff.delta" = "yellow"
+"diff.delta" = "light-blue"
+"diff.delta.moved" = "blue"
 "diff.minus" = "light-red"
 
-"ui.gutter" = "gray" 
-"info" = "light-blue" 
-"hint" = "gray" 
-"debug" = "gray"  
-"warning" = "yellow" 
-"error" = "light-red" 
+"ui.gutter" = "gray"
+"info" = "light-blue"
+"hint" = "light-gray"
+"debug" = "light-gray"
+"warning" = "light-yellow"
+"error" = "light-red"
 
-"diagnostic" = { modifiers = ["underlined"] }
-"diagnostic.info" = { fg = "light-blue", modifiers = ["underlined"] }
-"diagnostic.hint" = { fg = "gray", modifiers = ["underlined"] }
-"diagnostic.debug" ={ fg ="gray",  modifiers = ["underlined"] }
-"diagnostic.warning" = { fg = "yellow", modifiers = ["underlined"] }
-"diagnostic.error" = { fg ="light-red", modifiers = ["underlined"] }
+"diagnostic.info" = { underline = { color = "light-blue", style = "dot" } }
```

Comment:
I think the documentation is currently incorrect about this underline style, it should be `dotted`: https://github.com/helix-editor/helix/blob/f323ffabcc5846f647e3ba1d3d2ff5dec95257bd/helix-view/src/graphics.rs#L323

Reply:
Yeah I think I mixed that up :sweat_smile:  I think the documentation was what I originally planning. We use `curl` and `line` so using the passive `dotted` and `dashed` seemed inconsistent. Perhaps this should be changed to `dashes` and `dots` in both documentation and implementation?

Reply:
I think they're ok as passive - it makes sense to me as "dotted line" or "dashed line". When in doubt I would say we should follow whatever kitty says they are but it looks like then we would need to change to "curly" too for consistency: https://sw.kovidgoyal.net/kitty/underlines/

I think that just updating the documentation to say "dotted" would be the best choice to minimize breaking changes